dev/tasks/build/builder.js

```diff
@@ -0,0 +1,236 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* jshint node: true */
+
+'use strict';
+
+module.exports = Builder;
+
+/**
+ * A CKEditor 5 release builder.
+ *
+ * @class Builder
+ */
+function Builder( target ) {
+	/**
+	 * The target directory where to create the build.
+	 *
+	 * **Warning**: if existing, this directory will be deleted before processing.
+	 *
+	 * @property {String}
+	 */
+	this.target = target || 'build';
+
+	/**
+	 * The temporary directory to use for build processing.
+	 *
+	 * **Warning**: if existing, this directory will be deleted before processing.
+	 *
+	 * @property {String}
+	 */
+	this.tmp = 'build_tmp';
+
+	/**
+	 * The list of tasks to be executed by the `build()` method. Each entry is an Array containing the name of the
+	 * method inside `tasks` to execute and the message to show to the end user when executing it.
+	 *
+	 * @property {Array}
+	 */
+	this.taskList = [
+		[ 'cleanUp', 'Cleaning the "' + target + '" directory...' ],
```

Comment:
Now (49b17a2) I've got:

```
Running "build" task
1. Cleaning the "undefined" directory...
```

BTW. I miss very much an information about where the build was created - where should I look for it now.